Tiltfile

```diff
@@ -174,6 +174,10 @@ for resource in manager_yaml:
     if resource["metadata"]["name"] == "capl-manager-credentials":
         resource["stringData"]["apiToken"] = os.getenv("LINODE_TOKEN")
         resource["stringData"]["dnsToken"] = os.getenv("LINODE_DNS_TOKEN")
+        if os.getenv("LINODE_URL"):
+            resource["stringData"]["LINODE_URL"] = os.getenv("LINODE_URL")
+        if os.getenv("LINODE_CA_BASE64"):
+            resource["stringData"]["SSL_CERT_DIR"] = "/tmp/linode-ca"
```

Comment:
I think this value can safely be set to LINODE_CA now that we have a newer version of linodego(I believe an earlier version broke the `LINODE_CA` env variable usage)